android/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,3 @@
 <resources>
-    <string name="app_name">CannabisApp</string>
+    <string name="app_name">DrCannabis</string>
```

Comment:
Can we rename it to `Dr. Cannabis`?

Reply:
It should be easy to change. @MPucheta ?